### PR TITLE
Makes add_attacklogs able to take non-mob targets

### LINKED ISSUE #17664: Projectile attack logs runtime
#### Steps to reproduce 
- Be alien empress
- shoot at anything that isn't a mob
```
x14 logging.dm,76: undefined variable /obj/machinery/door/firedoor/border_only/var/ckey
  proc name: add attacklogs (/proc/add_attacklogs)
  usr: The alien empress (harukasuka) (/mob/living/simple_animal/hostile/alien/queen/large)
  usr.loc: The floor (232, 250, 1) (/turf/simulated/shuttle/floor)
  src: null
  call stack:
  add attacklogs(the alien empress (/mob/living/simple_animal/hostile/alien/queen/large), Firelock South (/obj/machinery/door/firedoor/border_only), "fires at", null, null, 1)
  the alien empress (/mob/living/simple_animal/hostile/alien/queen/large): TryToShoot(the plating (232,252,1) (/turf/simulated/floor/plating), Firelock South (/obj/machinery/door/firedoor/border_only))
  the alien empress (/mob/living/simple_animal/hostile/alien/queen/large): OpenFire(Firelock South (/obj/machinery/door/firedoor/border_only))
```

@Pathid 

#### Specific information for locating
#17451

#### Issue bingo

- [x] Issue could be reproduced at least once
- [x] Issue could be reproduced by different players
- [x] Issue could be reproduced in multiple rounds
- [x] Issue happened in a recent (less than 7 days ago) round
- [x] [Couldn't find an existing issue about this](https://github.com/d3athrow/vgstation13/issues)

### PR DESCRIPTION
**Pull request by Pathid**
Closes #17664
~~This also makes player simple mobs shooting not adminwarn as a followup to #17451, since I assume they don't want their chat to be a million Mecha Hitler bullets.~~

### CODE REVIEW

**Comment by Pathid** [comment]
Just found out Mecha Hitler in particular goes around the logging and I'm not sure exactly how. Anyway I still think it's a good idea to not have them be adminwarn.

File: code/modules/mob/living/simple_animal/hostile/hostile.dm [hunk 1]
```diff
@@ -301,7 +301,7 @@
 		if(ranged_message)
 			visible_message("<span class='warning'><b>[src]</b> [ranged_message] at [target]!</span>", 1)
 		if(ckey)
-			add_attacklogs(src, target, "[ranged_message ? ranged_message : "shot"] at")
+			add_attacklogs(src, target, "[ranged_message ? ranged_message : "shot"] at", admin_warn = FALSE)
```
Comment (Probe1) [inline code/modules/mob/living/simple_animal/hostile/hostile.dm]: Why false?

Comment (Pathid) [reply code/modules/mob/living/simple_animal/hostile/hostile.dm]: I don't know how useful reporting every bullet to admins is, especially when this kind of thing is usually from a bus. But I can change it back, sure.

Comment (Probe1) [reply code/modules/mob/living/simple_animal/hostile/hostile.dm]: It'd be for the best.  We can't predict all the times a player shooting another player will be good or bad.

**Pull request merged**